src/App.tsx

```
@@ -1,10 +1,106 @@
+    console.error("Data is null"); // ❌ Console.error
+    return false;
+  }
+  if (data.length < 5) {
+    // ❌ Magic number 5
+    return false;
+  }
+  return true;
+}
+
+// ❌ No early return pattern
+function checkUser(user: any) {
```

ESLint

[error] 'checkUser' is defined but never used.

(@typescript-eslint/no-unused-vars)